src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -425,6 +425,15 @@ public static string ConfirmDroppingExcelColumns(string listOfExcelColumns)
             return sb.ToString();
          }
 
+         public static string SheetsNotFound(List<string> listOfSheetNames)
+         {
+            var sb = new StringBuilder();
+            sb.AppendLine("The following excel sheets were not found in the file and will not be imported ");
+            sb.AppendLine();
+            listOfSheetNames.ForEach(item => sb.Append(item + "\n"));
```

Comment:
any reason not to use ?AppendLine. Why are you adding the \n by hand?

Reply:
corrected